core/src/main/java/tc/oc/pgm/PGMConfig.java

```diff
@@ -183,6 +184,7 @@ public final class PGMConfig implements Config {
     this.showProximity = parseBoolean(config.getString("ui.proximity", "false"));
     this.showSideBar = parseBoolean(config.getString("ui.sidebar", "true"));
     this.showTabList = parseBoolean(config.getString("ui.tablist", "true"));
+    this.resizeTabList = parseBoolean(config.getString("ui.resize", "true"));
```

Comment:
Let's set the default to be `false`, since this will log a warning if `ProtocolLib` is not loaded.

Reply:
Will do